Medvedev homework 7 arrays tasks

jacsmedvedev opened the pull request:
Как то так.  Только проблема - даже после изменения кодировки все равно кириллица некорректно отображается.

jacsmedvedev committed:
Коммичу и пушу !!!!

jacsmedvedev committed:
Переименовал

jacsmedvedev committed:
Первая часть задач.

strazhkoe reviewed (changes requested):
Нужно пофиксить кодировку, прежде чем сливать в мастер. Также разбей, пожалуйста, задачи на файлы. Так их будет проще проверить и не будет закоментированного кода.

jacsmedvedev committed:
11.8 Заполнить массив из восьми элементов так, чтобы при просомтре слева
направо они образовывали:
а) убывающую последовательность;

jacsmedvedev committed:
11.8 

Заполнить массив из восьми элементов так, чтобы при просомтре слева
направо они образовывали:
б) возрастающую последовательность;

jacsmedvedev committed:
11.12 a)

Заполнить массив:
а) двадцатью первыми натуральными числами, делящимися нацело на 13 или
17 и находящимися в интервале , левая граница которого 300.

jacsmedvedev committed:
11.15 a)

Дан массив. Составить программу:
 а) расчета квадратного корня из любого элемента массива

jacsmedvedev committed:
11.15 б)

Дан массив. Составить программу:
 б) расчета среднего арифметического двух любых элементов массива

jacsmedvedev committed:
11.17

Дан массив. Все его элементы:
а) увеличить в 2 раза
б) уменьшить на число А
в) разделить на первый элемент

jacsmedvedev committed:
Update Medvedev_task_11_17_abc.java

Здесь 3 условия в одном файле. Они пустяковые. Решил не создавать отдельно для каждого файл.

strazhkoe commented on src/medvedev_Homework_7_arrays_tasks/Medvedev_task_11_15_a.java:
```diff
@@ -0,0 +1,24 @@
+
+		System.out.println(Arrays.toString(array));
+		Scanner x = new Scanner(System.in);
+		System.out.println("Enter the array element number --- ");
```
Юзеру нужно сказать, в каком диапазоне вводить числа. Затем нужно сделать проверку на то, что он ввел правильно

strazhkoe commented on src/medvedev_Homework_7_arrays_tasks/Medvedev_task_11_15_a.java:
```diff
@@ -0,0 +1,24 @@
+		int numOfDigit = x.nextInt();
+		double zzz = Math.pow(array[numOfDigit-1],2);
+		System.out.println(zzz);
+	}
```
сканнер не закрыт

strazhkoe commented on src/medvedev_Homework_7_arrays_tasks/Medvedev_task_11_17_abc.java:
```diff
@@ -0,0 +1,30 @@
+		// System.out.println("All numbers of array reduce by A.");
+		// System.out.println("All numbers of array to divide by first element
+		// of array.");
+		// final double firstElement = array[0];
```
Удали закомментированный код

strazhkoe commented on src/medvedev_Homework_7_arrays_tasks/Medvedev_task_11_15_b.java:
```diff
@@ -0,0 +1,29 @@
+
+		System.out.println(Arrays.toString(array));
+		Scanner x = new Scanner(System.in);
+		System.out.println("Enter the array first element number --- ");
```
Выведи диапазон допустимых значений пользователю и проверь правильность ввода

strazhkoe reviewed (commented):
Добавь, пожалуйста, в комменты условия решаемых задач

jacsmedvedev committed:
Update Medvedev_task_11_17_abc.java

jacsmedvedev committed:
Revert "Update Medvedev_task_11_17_abc.java"

This reverts commit 56ffb3cb87bcdc9d783bcec5854182e1af7e1307.

jacsmedvedev committed:
Merge branch 'medvedev_Homework_7_arrays_tasks' of https://github.com/Akros2016/ort-dnepr-javacore-medvedev into medvedev_Homework_7_arrays_tasks

jacsmedvedev committed:
Проверка Кириллицы

jacsmedvedev committed:
Проверка Кириллицы

jacsmedvedev committed:
Update Medvedev_Cyrillic_check.java

jacsmedvedev committed:
Delete Medvedev_Cyrillic_check.java

jacsmedvedev committed:
11.15 а) Исправленное 

Дан массив. Составить программу:
 а) расчета квадратного корня из любого элемента массива

jacsmedvedev committed:
11.15 а) Исправлено

Дан массив. Составить программу:
 а) расчета квадратного корня из любого элемента массива

jacsmedvedev committed:
Merge branch 'medvedev_Homework_7_arrays_tasks' of https://github.com/Akros2016/ort-dnepr-javacore-medvedev.git into medvedev_Homework_7_arrays_tasks

jacsmedvedev committed:
11.15 б) Исправлен

Дан массив. Составить программу:
 б) расчета среднего арифметического двух любых элементов массива

jacsmedvedev committed:
11.40 

Дан массив. Вывести на экран сначало его неотрицательные , а потом
отриц. элементы.

jacsmedvedev committed:
11.53 а)

Дан массив целых чисел.
а) Все элементы кратные 10, заменить нулем.

strazhkoe committed:
Some fixes.

strazhkoe committed:
Removed old files.

strazhkoe commented:
Внес некоторые исправления и солью в мастер. Далее создавай по ветке и по реквесту на задачу
